doc/source/ecosystem.rst

```diff
@@ -575,3 +575,75 @@ Library            Accessor     Classes                              Description
 .. _composeml: https://github.com/alteryx/compose
 .. _datatest: https://datatest.readthedocs.io/
 .. _woodwork: https://github.com/alteryx/woodwork
+
+
+Development tools
+----------------------------
+
+`pandas-stubs <https://github.com/VirtusLab/pandas-stubs>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+pandas doesn't expose any type information to the user by itself.
```

Comment:
this is not true there is lots of type information, explain how this is different.

Reply:
is this better? `While pandas repository is partially typed, the package itself doesn't expose this information for external use. `